lib/ddtrace/profiling/scheduler.rb

```diff
@@ -12,7 +12,12 @@ class Scheduler < Worker
       include Workers::Polling
 
       DEFAULT_INTERVAL_SECONDS = 60
-      MIN_INTERVAL_SECONDS = 0
+      MINIMUM_INTERVAL_SECONDS = 0
+
+      # Profiles with duration less than this will not be reported
+      PROFILE_DURATION_THRESHOLD_SECONDS = 1
```

Comment:
Wouldn't 1 second be too high of a threshold for short running processes, like a one-off Rake task?

I only ask because 1 second is being hard-coded into this constant, so there's no way around it for short-running jobs with this change.

Reply:
Yes, that's true. My concern here is that the current profiler UX is built for long-running profiles; it's even unclear when you're in "search" that you're looking at a really short profile vs a regular 1-minute one.

If we think that 1 second is still too much, I'd rather lower it for everyone. I was hoping not to make it configurable until we got some clear requests/feedback that this was needed.

Reply:
I'd say go for the minimum amount of time that you'll think ensures a good balance here.
If it's 1 second, I'm ok with that.

Reply:
👍. Happy to revise at any time.